pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go

```diff
@@ -101,7 +101,8 @@ func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *res
 		infra.Status.PlatformStatus.Type == configv1.NutanixPlatformType ||
 		infra.Status.PlatformStatus.Type == configv1.VSpherePlatformType ||
 		infra.Status.PlatformStatus.Type == configv1.BareMetalPlatformType ||
-		infra.Status.PlatformStatus.Type == configv1.OpenStackPlatformType {
+		infra.Status.PlatformStatus.Type == configv1.OpenStackPlatformType ||
+		infra.Status.PlatformStatus.Type == configv1.NonePlatformType {
```

Comment:
please add a comment in the code and in the commit message explaining why these tests are skipped on this platform

Reply:
Added comment.